cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,37 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
+  const user = {
+    username: 'Flood',
+    password: 'qwert!Q12345',
+    textForSearching: 'Speaking JavaScript',
+    title: 'Speaking JavaScript',
```

Comment:
```suggestion
    textForSearching: 'Speaking JavaScript',
    title: 'Speaking JavaScript',
```
why do you need two the same variables but with the different name? left 1 variable and reuse it